### ISSUE TITLE
memory corruption when connecting a signal outlet to a non-signal inlet

### ISSUE THREAD

**Issue by umlaeute** [pure-data/pure-data#247]
tm reported this:

 1. turn `DSP ON`
 2. open a new window (with auto-patching enabled), and in the window do:
 3. <kbd>Ctrl</kbd>+<kbd>1</kbd> and type `*~`
 4. <kbd>Ctrl</kbd>+<kbd>1</kbd> and type `sig~`
 5. <kbd>Ctrl</kbd>+<kbd>e</kbd> (to create the second object)
 6. crash!

**Comment by umlaeute** [comment]
this was reported on OSX (or W32?).

I haven't been able to reproduce this on linux.
However, running the above cascade through `valgrind` revealed a number of memory access violations.

The problem can be reproduced with this minimal patch:
~~~pd
#N canvas 3 59 450 300 10;
#X obj 108 60 osc~;
#X obj 108 82 sig~;
#X connect 0 0 1 0;
~~~

As soon as audio is turned on, this triggers
> consistency check failed: ugen_connect osc~ sig~ 0 0 (1 0)
(or a crash, on less blessed systems).

The problem seems to be that connecting a signal outlet with a DSP-object that only has a control inlet (e.g. `[sig~]`) does not trigger the "signal outlet connected to non-signal inlet" codepath.

**Comment by umlaeute** [comment]
i just pushed a fix to the`update/0.48.0` branch
